File: u3u2/Assets/Scripts/app/net/c2s/CGCorpswarInfo.cs
using System;
using System.IO;
namespace app.net
{

/**
 * 请求帮派竞赛信息
 * 
 * @author CodeGenerator, don't modify this file please.
 */
public class CGCorpswarInfo :BaseMessage
{
	
	
	public CGCorpswarInfo ()
	{
	}
	
	
	protected override void ReadImpl() 
	{
		return;
	}
	
	protected override void WriteImpl() 
	{

	}
	
	public override short GetMessageType()
	{
		return (short)MessageType.CG_CORPSWAR_INFO;
	}
	
	public override string getEventType()
	{
		return "";
	}
	
	}
}

File: server/当前版本/server/zlj/game_server/src/com/imop/lj/gameserver/corps/msg/CGCorpswarInfo.java
package com.imop.lj.gameserver.corps.msg;

import com.imop.lj.core.msg.MessageType;
import com.imop.lj.gameserver.common.msg.CGMessage;
import com.imop.lj.gameserver.corps.handler.CorpsHandlerFactory;

/**
 * 请求帮派竞赛信息
 * 
 * @author CodeGenerator, don't modify this file please.
 */
public class CGCorpswarInfo extends CGMessage{
	
	
	public CGCorpswarInfo (){
	}
	
	
	@Override
	protected boolean readImpl() {


		return true;
	}
	
	@Override
	protected boolean writeImpl() {

		return true;
	}
	
	@Override
	public short getType() {
		return MessageType.CG_CORPSWAR_INFO;
	}
	
	@Override
	public String getTypeName() {
		return "CG_CORPSWAR_INFO";
	}


	@Override
	public void execute() {
		CorpsHandlerFactory.getHandler().handleCorpswarInfo(this.getSession().getPlayer(), this);
	}
}